homeassistant/components/nx584/alarm_control_panel.py

```diff
@@ -58,30 +50,48 @@ async def async_setup_platform(hass, config, async_add_entities, discovery_info=
 
     url = f"http://{host}:{port}"
 
-    device = NX584Alarm(url, name)
+    try:
+        alarmclient = client.Client(url)
```

Comment:
Please separate words with underscore and rename the variable to `alarm_client`.